src/comps/update/batch.js

```diff
@@ -48,6 +49,11 @@ const batchUpdateRound = async ({
         await updateDb(predictoor_data, "predictoor_data", roundNumber)
     }
 
+    if (predictoor_summary) {
+        await dropTable("predictoor_summary", roundNumber)
+        await updateDb(predictoor_summary, "predictoor_summary", roundNumber)
+    }
+
```

Comment:
This looks great, however I think we're missing the implementation of `updatePredictoorSummary()`, unless this is done at the df-py level and comes for free.

Something that looks like `updateRewardsSummary` and creates an aggregate/summary table.